### DISCUSSION TITLE
[제페토] 커스텀 월드 제작 - 단순한 2D 미니게임이 개발이 가능할까요?

### DISCUSSION THREAD

**Discussion by ellejou76** [naverz/zepeto-studio-kor#490] [5. Q&A]
안녕하세요. 

제페토 월드를 기획/개발 하려고 하는데.
월드 내에서  다른 친구들과는 플레이는 아니고 

자기 혼자, 숨은그림찾기 같은 단순한 2D 게임도 개발이  제작이 가능할까요?
Unity 3d 로는 개발이 가능할 것 같은데  월드 제작시 그렇게 개발을 해도 심사 등 문제가 없을까요?

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 공모전을 위한 작품이라면 아바타가 어떻게든 표현이 되는 것이 원칙입니다.
그냥 일반적인 출시가 목적이라면 가능하며 
이미 관련 게임도 제페토 내에 출시가 되어 있습니다 😄
